app/serializers/project_status_serializer.rb

```diff
@@ -29,4 +30,8 @@ def name
   def canonical_name
     object.name
   end
+
+  def repository_license
+    object.repository&.license
```

Comment:
what is the repository license, is this from GitHub analyzing the repo?

Reply:
Yeah I think this comes from the API for the supported repository hosts. I'm having a hard time telling what the project normalized_licenses is actually returning, but I think this will be helpful.